bins/reth-crawler/src/crawler/factory.rs

```diff
@@ -49,7 +49,7 @@ impl CrawlerFactory {
         }
     }
 
-    pub async fn make(&self) -> CrawlerService {
-        CrawlerService::new(self.discv4.clone(), self.dnsdisc.clone(), self.key).await
+    pub async fn make(&self, sql_db: bool) -> CrawlerService {
```

Comment:
eventually would be nice to remove the use of bool flags like this (`sql_db` works for now but in the future if there are more than 2 possibilities?)

probably makes sense to use an enum - and even then it'd be nice to see if there's a better way to do this (without passing in this flag at all, maybe creating the DB separately and passing it as a generic param to make?)